### DISCUSSION TITLE
set defaultClient in configuration

### DISCUSSION THREAD

**Discussion by flipvh** [scalar/scalar#2074] [Ideas]
Hi all,
We would love to be able to set a defaultClient. the reason is that our likely target audience is a TS/JS developer, so instead of cUrl we would like to show a Node option by default. Is it correct that this currently is not possible or did I overlook this option? Would others also benefit from setting this default? 

Thnks!

**Comment by flipvh** [comment]
Its related to this discussion to have more control over de clients: https://github.com/scalar/scalar/discussions/807

**Comment by hanspagel** [comment]
Yes! We need this. I’ve created an issue: https://github.com/scalar/scalar/issues/2075
